apps/avifenc.c

```diff
@@ -1117,6 +1118,8 @@ int main(int argc, char * argv[])
         } else if (!strcmp(arg, "-V") || !strcmp(arg, "--version")) {
             avifPrintVersions();
             goto cleanup;
+        } else if (!strcmp(arg, "-n") || !strcmp(arg, "--no-overwrite")) {
```

Comment:
Please add a description of the flag in the syntax() function. Also add a line in CHANGELOG.md